### PR TITLE
Restyled the retry button on about:error

### LINKED ISSUE #2147: Retry button on about:error
**Describe the issue you encountered:** The retry button on ``about:error`` is not well styled.

I'm pushing a PR to fix this into:

### PR DESCRIPTION
**Pull request by luixxiul**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Ran `git rebase -i` to squash commits if needed.

Fixes #2147

Also refactored button.less, adding a class "actionButton" to replace the class of allowScriptsOnce button with it, which was introduced with #1564.

Image:

### CODE REVIEW

**Comment by luixxiul** [comment]
``about:certerror`` needs rewriting due to this change. I'm working on it.

**Comment by luixxiul** [comment]
If everything is ok I will rebase commits and and push them again.

File: app/extensions/brave/locales/en-US/error.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-errorReload=Retry {{url}}
```
Comment (luixxiul) [inline app/extensions/brave/locales/en-US/error.properties]: I think repeating the address is too much

File: less/button.less [hunk 2]
```diff
@@ -34,51 +45,56 @@ span.browserButton {
...
+
+  &.subtleButton {
+    background-color: #ccc;
+    font-size: 14px;
+  }
+
+  &.wideButton {
+    display: block;
+    margin-right: 0;
+    margin-left: 0;
```
Comment (luixxiul) [inline less/button.less]: To remove margins (https://github.com/brave/browser-laptop/pull/2149/files#diff-fca7d0eef4fe1430dbdc3733b1fc5fdaR27)

**Comment by diracdeltas** [comment]
lgtm, thanks!

**Comment by luixxiul** [comment]
Working on #2205 I noticed that the margin around the button was not specified well.

Here margin-left should be null. I'm pushing a PR to fix it.

**Pull request merged**